Agent/intents/Default Welcome Intent_usersays_en.json

```diff
@@ -1,9 +1,9 @@
 [
   {
-    "id": "a9ec9fa8-550c-4d65-972a-50a0ae9ee9b0",
+    "id": "2bd53557-3af8-4694-a80d-3489f9a3fdbc",
     "data": [
       {
-        "text": "just going to say hi",
+        "text": "hello hi",
```

Comment:
?

Reply:
Ahh, this must be autogenerated or smt, right?

Reply:
yes